js/avatarCreation.js

```diff
@@ -0,0 +1,38 @@
+
+
+//$('#player1').setAttribute('x', '150' ) ;
+
+var playerAx = 150;
+var playerAy = 150;
+var playerAHeight = 50;
+var playerAWidth = 50;
+var playerAStartX = 150;
+var playerAStartY = 150;
+var speed = 50;                     //change in x. As this is based on setInterval rate = speed
+
+
+
+var playerBx = 550;
+var playerBy = 550;
+var playerBHeight = 50;
+var playerBWidth = 50;
+var playerBStartX = 550;
+var playerBStartY = 550;
+
```

Comment:
You can separate `var` by commas after the first one. Group variables with the first `var`

```
var something = 'yeah',
      anodaVar = 'nope'

var anotherSetOfVars = 'here',
      init = 'fun'